tasks/cephfs/test_volume_client.py

```diff
@@ -131,6 +131,26 @@ def test_lifecycle(self):
         self.mounts[2].mount(mount_path=mount_path)
         self.mounts[2].write_n_mb("data.bin", 1)
 
+        # Our data should stay in particular rados namespace
+        from ceph_volume_client import CephFSVolumeClient, VolumePath
+        vc = CephFSVolumeClient("manila", self.mount_b.config_path, "ceph")
+        vc.connect()
+        pool_name = vc._get_ancestor_xattr(vc._get_path(VolumePath(group_id, volume_id)), "ceph.dir.layout.pool")
+        ioctx = vc.rados.open_ioctx(pool_name)
+        NS_PREFIX = "fsvolumens_"
+        namespace = "{0}{1}".format(NS_PREFIX, volume_id)
+        ioctx.set_namespace(namespace)
+        it = ioctx.list_objects()
```

Comment:
See the Filesystem.rados() function which wraps the rados command line tool, this will let you run it remotely.  Use this to list objects in the namespace (similar to how it's used in Filesystem._enumerate_data_objects)